packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts

```diff
@@ -879,7 +878,7 @@ export const shortCodeMap = {
     "0x457": "Index must be less than length.",
     "0x458": "Count must be non-negative.",
     "0x459": "The node does not exist.",
-    "0x45a": "`transactionCheckout` is required to edit the EditableTree",
+    "0x45a": "transactionCheckout` is required to edit the EditableTre",
```

Comment:
Ah, I think I know what happened... the policy check handler tried to clean up the value because it thought the backticks were the remnants from an interpolated string. That was useful when I created the mapping file, but since the handler now rejects interpolated strings as arguments to `assert()` in the first place, it shouldn't need to do this cleanup anymore, I'll look into it to remove that step and fix this value in the map.

It seems it only showed up now because it was the previous release that made this string into a hexcode + comment, and only now did the handler try to clean it up.